src/metatensor/models/utils/model_io.py

```diff
@@ -0,0 +1,65 @@
+import importlib
+from typing import Dict, List
+
+import torch
+
+
+def save_model(
+    arch_name: str,
+    model: torch.nn.Module,
+    hypers: Dict,
+    all_species: List[int],
+    path: str,
+) -> None:
+    """Saves a model to a file, along with all the metadata needed to load it.
+
+    Parameters
+    ----------
+        arch_name (str): The name of the architecture.
+
+        model (torch.nn.Module): The model to save.
+
+        hypers (Dict): The hyperparameters used to train the model.
+
+        all_species (List[int]): The list of all species that the model can handle.
+
+        path (str): The path to the file.
+    """
+    torch.save(
+        {
+            "name": arch_name,
+            "model": model.state_dict(),
+            "hypers": hypers,
```

Comment:
should we call it `model_hypers`, since the have so many hyper parameters...

Reply:
True